### DISCUSSION TITLE
How can I let capsule character look at vector3 position?

### DISCUSSION THREAD

**Discussion by 0x5143** [bepu/bepuphysics2#264] [Q&A] [answered]
I need something like `Transform.LookAt` in Unity.

**Comment by RossNordby** [comment, marked as answer]
1. Matrix4x4 has some helper functions that you could use. That's a part of System.Numerics. 
2. I wouldn't recommend modifying character body orientation just for yaw, though. Given a capsule's rotational invariance, it doesn't do anything. Instead, you can just set the character's view direction and such, and then gameplay logic and graphics can use that information.
